forms-flow-api/Dockerfile

```diff
@@ -6,6 +6,14 @@ LABEL Name="formsflow"
 
 WORKDIR /forms-flow-api/app
 
+# Set build arguments for workers and threads
```

Comment:
@kiran-aot This would make these configuration values to be provided during build time right ? What I was thinking was to keep this as runtime variable and not be a build time dependency. This would let you change the variables runtime based on your computing changes.

Reply:
Updated the Dockerfile and  removed from Dockerfile to avoid build dependency